lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1626,11 +1626,17 @@ func (w *WalletKit) FundPsbt(_ context.Context,
 			return nil, fmt.Errorf("unknown change output type")
 		}
 
+		maxFeeRatio := chanfunding.DefaultMaxFeeRatio
+
+		if float64(req.MaxFeeRatio) != float64(0) {
```

Comment:
Since everything is `float64` now, none of these casts in the condition or assignment are needed.